models/demos/falcon7b/tests/test_perf_falcon.py

```diff
@@ -469,16 +469,14 @@ def run_perf_wh_bare_metal(
     ):
         if model_config_str == "BFLOAT16-L1_SHARDED" and kv_cache_len == 2047:
             pytest.skip(f"kv_cache_len={kv_cache_len} does not fit with L1_SHARDED")
-        if model_config_str == "BFLOAT16-L1_SHARDED" and llm_mode == "prefill":
-            pytest.skip(f"prefill does not support L1_SHARDED")
```

Comment:
The l1 sharded config was originally designed as the optimized decode config, should we be using that for prefill? Same question for all the other tests that have been modified to use l1 sharded for prefill.

Reply:
I reverted back to DRAM to avoid introducing confusion; We actually have one mode only for optimized prefill, so we will clean this up afterwards.